app/src/main/java/tech/pauly/findapet/data/models/Sex.java

```diff
@@ -7,11 +7,13 @@
 import tech.pauly.findapet.R;
 
 public enum Sex {
-    U(0, R.string.unknown, null),
-    M(1, R.string.male, "M"),
-    F(2, R.string.female, "F");
+    MISSING(0, R.string.missing, null),
+    MALE(1, R.string.male, "M"),
+    FEMALE(2, R.string.female, "F"),
+    UNKNOWN(3, R.string.unknown, "U");
```

Comment:
Note: Petfinder can return "U" (for unknown I assume?). I'm not exposing it to users as it's not really useful. But I need to make a distinction between that and Missing, which is the option that Filter objects use as default since it sends null to the backend so M, F, and U are all returned.